File: 2129-capitalize-the-title/capitalize.ts
function capitalizeTitle(title: string): string {
    let newTitle=[]
    title.split(' ').forEach(word=>{
    word.length>2 ? newTitle.push(word[0].toUpperCase() + word.toLowerCase().slice(1,)) : newTitle.push(word.toLowerCase())
        })
    return newTitle.join(' ')
};

File: 2129-capitalize-the-title/capitalize.rb
# @param {String} title
# @return {String}
def capitalize_title(title)
    newTitle = []
    title.downcase.split(' ').map{|word| 
        if word.size >2 
            newTitle.push(word.capitalize)
        else
            newTitle.push(word)
        end 
    }
    newTitle.join(' ')
    
    # alternative much faster solution
    # title.split.map {|word| word.length < 3 ? word.downcase : word.capitalize}.join(" ")
end